src/fontra/views/editor/panel-transformation.js

```diff
@@ -450,6 +579,452 @@ export default class TransformationPanel extends Panel {
     this.update();
   }
 
+  _getObjectBounds(layerGlyphController, selection) {
+    if (selection.startsWith("contour")) {
+      const contour = selection.split("/")[1];
+      const points = contour.split(",").map((pointIndex) => `point/${pointIndex}`);
+      return layerGlyphController.getSelectionBounds(points);
+    }
+
+    return layerGlyphController.getSelectionBounds([selection]);
+  }
+
+  _getSortedObjects(
+    layerGlyphController,
+    points,
+    contours,
+    components,
+    distributeDirection
+  ) {
+    let objects = [];
+    for (const pointIndex of points) {
+      objects.push(`point/${pointIndex}`);
+    }
+    for (const contour of contours) {
+      objects.push(`contour/${contour}`);
+    }
+    for (const componentIndex of components) {
+      objects.push(`component/${componentIndex}`);
+    }
+
+    let objectsSorted = objects.sort(
+      (a, b) =>
+        this._getObjectBounds(layerGlyphController, a).yMin -
```

Comment:
I need to sort the 'objects' somehow to fix the issue we had in yesterdays scrum demo with the components which were (wrongly) reordered.
I thought it makes sense to figure our the order from left (=xMin) to right and from baseline/desender (=yMin) to top . The center would not be helpful at that point. It's only to figure out the current order of the object selected.  
The bug has been fixet with this.

Reply:
I understand that this fixes the ordering issue, I'm just arguing about the exact sorting criterium. Given that the direction (left to right vs right to left / bottom to top vs top to bottom) has no intrinsic meaning (one is not better than the other, just different), I'd argue taking the center is perhaps more "fair": it would give the same result even if the whole collection of shapes were mirrored (but mirrored).

Anyway, minor point.